operators/tutorials/authentication/basic-auth-access-control.md

```diff
@@ -2,11 +2,11 @@
 description: Set up BasicAuthAccessControl for access to controller and broker
 ---
 
-# Basic auth access control
+# BasicAuthAccessControl
 
-## Set up tokens and user credentials
+# Tokens and User Credentials
```

Comment:
Was this intentionally changed from sentence case/imperative back?

Reply:
No, absolutely not! I'm not sure what's up with this file. I didn't touch it. :/

Reply:
I found an older commit with the proper sentences case and imperative forms and copied/pasted into this. This is the only change I'm making to this file in this PR.